config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get '/ingredients', to: 'ingredients#index'
+  get '/ingredients/:id', to: 'ingredients#show'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get '/recipes', to: 'recipes#index'
+  get '/recipes/:id', to: 'recipes#show'
 end
```

Comment:
Besides the extra routes that you have that weren't needed, you have good restful routes here.

I do also note that you have a `RecipeIngredientsController` but no routes going to it. That also proves that this controller could probably be safely removed.